translations/values-ja/strings.xml

```diff
@@ -63,7 +63,7 @@
     <string name="password">パスワード</string>
     <string name="cellphone_number">電話番号</string>
     <string name="exit_app">閉じる</string>
-    <string name="alipay">Alipay</string>
+    <string name="alipay">アリペイ (Alipay)</string>
```

Comment:
Thanks. Should replace the symbol here with parentheses commonly used in Japanese?

Reply:
Yes